cypress/integration/SEP.ts

```diff
@@ -616,725 +594,673 @@ context('SEP reviews and meeting component tests', () => {
 
-        cy.get('@timeAllocation').type('-1').blur();
-        cy.contains('Must be greater than or equal to');
+      cy.get('@timeAllocation').type('-1').blur();
+      cy.contains('Must be greater than or equal to');
 
-        cy.get('@timeAllocation').clear().type('987654321').blur();
-        cy.contains('Must be less than or equal to');
+      cy.get('@timeAllocation').clear().type('987654321').blur();
+      cy.contains('Must be less than or equal to');
 
-        cy.get('@timeAllocation').clear().type('9999');
-        cy.get('[data-cy="save-time-allocation"]').click();
+      cy.get('@timeAllocation').clear().type('9999');
+      cy.get('[data-cy="save-time-allocation"]').click();
 
-        cy.finishedLoading();
+      cy.finishedLoading();
 
-        cy.contains('9999 (Overwritten)');
+      cy.contains('9999 (Overwritten)');
 
-        cy.closeModal();
-        cy.contains('9999');
+      cy.closeModal();
+      cy.contains('9999');
 
-        cy.reload();
-        cy.contains('Meeting Components').click();
-        cy.get('[title="Show proposals"]').click();
+      cy.reload();
+      cy.contains('Meeting Components').click();
+      cy.get('[title="Show proposals"]').click();
 
-        cy.get('[title="View proposal details"]').click();
+      cy.get('[title="View proposal details"]').click();
 
-        cy.get('[data-cy="edit-sep-time-allocation"]').click();
-        cy.get('@timeAllocation').should('have.value', '9999');
-        cy.get('@timeAllocation').clear();
-        cy.get('[data-cy="save-time-allocation"]').click();
+      cy.get('[data-cy="edit-sep-time-allocation"]').click();
+      cy.get('@timeAllocation').should('have.value', '9999');
+      cy.get('@timeAllocation').clear();
+      cy.get('[data-cy="save-time-allocation"]').click();
 
-        cy.finishedLoading();
+      cy.finishedLoading();
 
-        cy.get('body').should('not.contain', '9999 (Overwritten)');
+      cy.get('body').should('not.contain', '9999 (Overwritten)');
 
-        cy.closeModal();
-      });
+      cy.closeModal();
+    });
 
-      it('should use SEP time allocation (if set) when calculating if they fit in available time', () => {
-        cy.login('officer');
-        cy.visit('/');
-        cy.contains('SEPs').click();
+    it('should use SEP time allocation (if set) when calculating if they fit in available time', () => {
+      cy.login('officer');
+      cy.visit(`/SEPPage/${createdSepId}`);
 
-        cy.contains(sep1.code).parent().find('button[title="Edit"]').click();
+      cy.contains('Meeting Components').click();
 
-        cy.contains('Meeting Components').click();
+      cy.finishedLoading();
 
-        cy.finishedLoading();
+      cy.get('[title="Show proposals"]').click();
 
-        cy.get('[title="Show proposals"]').click();
+      cy.get(
+        '[data-cy="sep-instrument-proposals-table"] tbody tr:last-child'
+      ).should('have.css', 'background-color', 'rgb(246, 104, 94)');
```

Comment:
I think it is better to add a "data-xxxxx" attribute indicating the state of the row, than tying it to the color.